lib/pact/init/templates/provider/pact_helper.erb

```diff
@@ -0,0 +1,8 @@
+require 'pact/provider/rspec'
+require 'service_consumers/provider_states_for_my_consumer'
+
+Pact.service_provider '<%= @names[:provider].strip %>' do
+  honours_pact_with '<%= @names[:consumer].strip %>' do
```

Comment:
Better to do the strip in the class, not the template. Templates should be as dumb as possible.